src/assets/scss/components/search.scss

```diff
@@ -0,0 +1,72 @@
+.search-input {
+  border: 0;
+  border-bottom: 2px solid rgb(51, 51, 51);
```

Comment:
There's a file called colours.scss in /src/assets/scss/variables/ which houses all the colours used throughout the project. This makes it easy to change a theme from one file instead of changing the colour value in each file. 

It is easy to understand how the colors.scss has been written. If you have any doubts, you can ping me. Please add any new colours to that file and then use the variable names to assign the colour in these individual files. Do it throughout this file and in the future.

Reply:
I have updated it now.